core/src/main/java/org/kiwix/kiwixmobile/core/utils/KiwixDialog.kt

```diff
@@ -114,6 +115,10 @@ sealed class KiwixDialog(
     object WifiOnly : YesNoDialog(
       R.string.wifi_only_title, R.string.wifi_only_msg
     )
+
+    object OpenInNewTab : YesNoDialog(
+      R.string.open_in_new_tab, R.string.open_in_new_tab
```

Comment:
the first parameter should probably be null, I don't think this dialog had a title